package/src/steps/FormView/BodyFormView.tsx

```diff
@@ -281,9 +281,9 @@ const BodyFormView: React.FC<BodyFormViewType> = (props) => {
                             ))
                             || ((field.type === 'boolean' && field.name === 'termsOfUse') && (
                                 <label key={i} className={`${stylesCommon.body__child} ${styles.terms}`}>
-                                    <input type="checkbox" checked={collected[field.name] ?? false} name={field.name} onChange={(e) => onChange(e.currentTarget.name, e.currentTarget.checked, e.currentTarget.type)} />&nbsp;I accept {
+                                    <input type="checkbox" checked={collected[field.name] ?? false} name={field.name} onChange={(e) => onChange(e.currentTarget.name, e.currentTarget.checked, e.currentTarget.type)} />&nbsp;{t('formView.termsAgreement')} {
                                         field.terms?.map<React.ReactNode>((term, i) => <a key={i} href={term.url} target='_blank' rel="noopener noreferrer">{term.humanName}</a>)
-                                            .reduce((acc, actual, i, arr) => [acc, i === arr.length - 1 ? ' and ' : ', ', actual])
+                                            .reduce((acc, actual, i, arr) => [acc, i === arr.length - 1 ? ` ${t('misc.and')} ` : ', ', actual])
```

Comment:
This could be simplified to `t('misc.and')` right?

Reply:
You mean to get rid of spaces? I've added the spaces before aned after the 'and', rather then including them in the translation value

Reply:
Smart 👍